### PR TITLE
Add Romanian translation

### PR DESCRIPTION
**Pull request by licaon-kter**
Translated with https://github.com/LonamiWebs/Stringlate

Do consider #7 too.

### CODE REVIEW

**Comment by licaon-kter** [comment]
Will this merge depend on whether you hire an employee that can speak this language or?

**Comment by licaon-kter** [comment]
Still discussing, eh? :-|

**Comment by licaon-kter** [comment]
Should I bother keeping this up to date or?

**Comment by Yonjuni** [comment]
As we are working on a big update to version 2.0 I would suggest to update the strings after the update, if that's okay. Probably, it's easiest that way.

File: app/src/main/res/values-ro/strings.xml [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    <string name="pref_title_use_wake_lock">Ține procesorul treaz</string>
+    <string name="pref_summary_use_wake_lock">Pe unele dispozitive procesorul poate adormii și oprește contorizarea pașilor. Activează asta pentru a menține treaz procesorul atunci când se contorizează pașii - poate duce la un consum ridicat al bateriei!</string>
+
+    <string name="pref_title_daily_step_goal">Obiectiv personal ca număr de pași zilnic</string>
+    <string name="pref_title_unit_of_length">Unitate de lungime</string>
+
+    <string-array name="pref_keys_unit_of_length">
+        <item>kilometrii</item>
+        <item>mile</item>
+    </string-array>
```
Comment (morckx) [inline app/src/main/res/values-ro/strings.xml:74]: invalid XML

File: app/src/main/res/values-ro/strings.xml [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    <string name="pref_summary_accelerometer_threshold">Aplicația folosește accelerometrul dispozitivului. Vă rugăm, pentru a putea optimiza detecția pașilor, ajustați sensibilitatea.</string>
+
+    <string name="pref_title_weight">Greutate (kg)</string>
+
+    <string name="pref_title_gender">Sex</string>
+
+    <string-array name="pref_keys_gender">
+        <item>femeie</item>
+        <item>bărbat</item>
+    </string-array>
```
Comment (morckx) [inline app/src/main/res/values-ro/strings.xml:93]: invalid XML

File: app/src/main/res/values-ro/strings.xml [hunk 3]
```diff
@@ -0,0 +1,235 @@
+<resources>
+    <string name="app_name">Pedometer</string>
+    <string name="app_name_long">Privacy Friendly Pedometer</string>
+
+    <!-- Menu Items-->
+    <string name="action_help">Ajutor</string>
+    <string name="action_about">Despre</string>
+    <string name="action_settings">Setări</string>
+    <string name="action_main">Pedometer</string>
+    <string name="action_navigation">Navigare</string>
```
Comment (morckx) [inline app/src/main/res/values-ro/strings.xml:10]: Some strings here don't exist (anymore).

File: app/src/main/res/values-ro/strings.xml [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+    <string-array name="pref_keys_accelerometer_threshold">
+        <item>foarte mare</item>
+        <item>mai mare</item>
+        <item>mare</item>
+        <item>normală</item>
+        <item>mică</item>
+        <item>mai mică</item>
+        <item>foarte mică</item>
+    </string-array>
+        <item>0.125</item>
```
Comment (morckx) [inline app/src/main/res/values-ro/strings.xml:109]: invalid XML

File: app/src/main/res/values-ro/strings.xml [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+    <string name="pref_title_notification_motivation_alert_texts">Mesaj notificare motivațională</string>
+    <string name="pref_title_notification_motivation_alert_criterion">Criteriu alertă motivațională</string>
+
+
+    <string-array name="pref_keys_notification_motivation_alert_criterion">
+        <item>mai puțin de 90% din obiectivul zilnic de pași</item>
+        <item>mai puțin de 70% din obiectivul zilnic de pași</item>
+        <item>mai puțin de 50% din obiectivul zilnic de pași</item>
+    </string-array>
+        <item>90</item>
```
Comment (morckx) [inline app/src/main/res/values-ro/strings.xml:140]: invalid XML

File: app/src/main/res/values-ro/strings.xml [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+    <string name="walking_mode_input_empty">Numele și lungimea pasului sunt obligatorii. Vă rugăm să le completați!</string>
+    <string name="hint_walking_mode_name">Nume</string>
+    <string name="hint_walking_mode_step_length">Lungime pas (metrii)</string>
+
+
+    <string-array name="pref_default_walking_mode_names">
+        <item>Mers</item>
+        <item>Alergat</item>
+    </string-array>
+        <item>0.5</item><!-- step length in meters -->
```
Comment (morckx) [inline app/src/main/res/values-ro/strings.xml:217]: invalid XML

**Review by morckx — CHANGES_REQUESTED** [review]
Some problems with XML validity need to be fixed. Also –  no surprise – some strings don't seem to exist anymore.

**Pull request closed without merging**